core/src/main/java/org/jboss/pnc/build/finder/protobuf/KojiArchiveInfoAdapter.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.build.finder.protobuf;
+
+import org.infinispan.protostream.annotations.ProtoAdapter;
+import org.infinispan.protostream.annotations.ProtoFactory;
+import org.infinispan.protostream.annotations.ProtoField;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.red.build.koji.model.json.util.KojiObjectMapper;
+import com.redhat.red.build.koji.model.xmlrpc.KojiArchiveInfo;
+
+/**
+ * Protostream adapter to be able to properly marshall / unmarshall KojiArchiveInfo.
```

Comment:
Really small nitpick. I don't think there should be spaces around `/`, although I am seeing multiple people use this style (I don't know if it varies by country or anything).